CHANGELOG.md

```diff
@@ -1046,9 +1046,13 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Fix missing Q4 21/22 option in external income by including "previous" year  as well as 
   the next ten. 
 
+## [release-109] 2022-05-18
+- Generates public (rather than pre-signed) URL for download link to fetch reports from S3 bucket
```

Comment:
In the changelog I'd try to answer the question "what does the user get out of this change" rather than a more technical "what have we done", but this is fine (and sometimes there isn't a particularly obvious answer)